pkgs/development/tools/gomod2nix/generic.nix

```diff
@@ -0,0 +1,45 @@
+{ version
+, src
+, modules
+}:
+{ stdenv
+, callPackage
+, go
+, lib
+, makeWrapper
+, installShellFiles
+, fetchFromGitHub
+}:
+
+let
+  builder = callPackage ./builder { };
+
+in
+builder.buildGoApplication {
+  pname = "gomod2nix";
+  inherit version modules src go;
+
+  allowGoReference = true;
+
+  subPackages = [ "." ];
+
+  nativeBuildInputs = [ makeWrapper installShellFiles ];
+
+  passthru = builder;
```

Comment:
This is just aliasing the `builder` attrset as passthru, it's totally intended.

Reply:
We should stay with the default passthru usage in case we ever refactor this and it becomes a problem. Changing it right now to the default usage is very easy but not when it is widely used.